test/test_torch.py

```diff
@@ -1232,7 +1232,7 @@ def test_masked_scatter(self):
                         src = torch.randn(num_copy - 1)
                         with self.assertRaises(RuntimeError):
                             dest.masked_scatter_(mask, src)
-            self.assertEqual(len(w), 27)
+            self.assertEqual(len(w), 20)
```

Comment:
I don't think the CUDA variant is tested.

https://github.com/pytorch/pytorch/blob/963f7629b591dc9750476faf1513bc7f1fb4d6de/test/test_torch.py#L1198-L1239

Reply:
Other tests for `masked_scatter`,

https://github.com/pytorch/pytorch/blob/963f7629b591dc9750476faf1513bc7f1fb4d6de/test/test_torch.py#L4590-L4600

https://github.com/pytorch/pytorch/blob/963f7629b591dc9750476faf1513bc7f1fb4d6de/test/test_torch.py#L5129-L5144